keyboards/pohjolaworks/louhi/config.h

```diff
@@ -0,0 +1,155 @@
+/*
+Copyright 2020 Erkki Halinen
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0x5057
+#define PRODUCT_ID   0x0001
+#define DEVICE_VER   0x0001
+#define MANUFACTURER Pohjola Works
+#define PRODUCT      Louhi
+
+/* Vial */
+#define VIAL_KEYBOARD_UID {0x7F, 0xD7, 0x1E, 0x05, 0xFC, 0x51, 0x44, 0xAE}
+#define VIAL_UNLOCK_COMBO_ROWS { 2 }
+#define VIAL_UNLOCK_COMBO_COLS { 0, 6 }
```

Comment:
these two arrays should contain the same number of elements, I've added a check to vial.c to make sure that is always the case